Lombiq.HelpfulExtensions/Manifest.cs

```diff
@@ -101,3 +101,14 @@
         "OrchardCore.Markdown",
     }
 )]
+
+[assembly: Feature(
+    Id = Workflows,
+    Name = "Lombiq Helpful Extensions - Workflows",
+    Category = "Workflows",
+    Description = "Adds useful workflow activities (e.g., generate reset password token).",
+    Dependencies = new[]
+    {
+        "OrchardCore.Workflows",
+    }
+)]
```

Comment:
As a continuation of my suggestion for `FeatureIds.cs`.

```suggestion
[assembly: Feature(
    Id = Workflows,
    Name = "Lombiq Helpful Extensions - Workflows",
    Category = "Workflows",
    Description = "Adds useful workflow activities (e.g., generate reset password token).",
    Dependencies = new[]
    {
        "OrchardCore.Workflows",
    }
)]

[assembly: Feature(
    Id = ResetPasswordActivity,
    Name = "Lombiq Helpful Extensions - Reset password workflow activity",
    Category = "Workflows",
    Description = "Adds generate reset password token activity.",
    Dependencies = new[]
    {
        Workflows,
        "OrchardCore.Users.ResetPassword",
    }
)]
```

Reply:
There are no other workflow-related features so I don't see the point of creating a Workflows features now. What do you think?

Reply:
I can agree, we can drop the `Workflows` feature.